src/main/java/io/github/cottonmc/cotton/gui/widget/WButton.java

```diff
@@ -65,7 +67,7 @@ public WButton(Text label) {
 	 * @param label the label
 	 * @since 2.2.0
 	 */
-	public WButton(Icon icon, Text label) {
+	public WButton(@Nullable Icon icon, @Nullable Text label) {
```

Comment:
The annos should also go on the other constructors (I think I forgot to add them at some point when adding the icon API)

Reply:
Yeah, I get the warning in IntelliJ - i will add it